site/contents/write_ups/2022/GLMBlogpost.md

```diff
@@ -0,0 +1,80 @@
+
+## Reflections
+
+We explored and discussed the pros and cons of these systems. Whilst some students were quick to claim there was no bias in these systems, most were quick to note that their bias was actually encoded into these systems by their choices of correlation coefficient. At least this bias was consistent -  someone evaluating teachers might evaluate the same two teachers differently depending on their mood that day (or how soon their lunchbreak was). 
+
+We had already touched on the importance of sample size in our running speed example, and this became more apparent. Some variables were (e.g. % attendance) per class. If a teacher teaches 5 classes, they have 5 data points to work with (most models had at least 5 input variables). For variables which were per student (e.g. grades, grade improvement), if we say 30 students to a class, we still only have 150 data points to work with – not enough. Even if these models were perfect predictors of how good a teacher is, the data going into them was insufficient to make them work well.
+
+Perhaps the most important was the lack of feedback loop. When you try and hit a target and you miss, you can adjust your aim to get closer next time. This is the case when your model is used to predict the outcome of an event. When you have a system that is evaluative instead of predictive (be it for teacher good-ness, how long of a prison sentence someone should serve or indeed what grade students should get if they can’t sit their exams due to a global pandemic), you can no longer see the target and so there is no built-in feedback loop. When you can’t see a target, evaluations often become more like judgements, some how given more value. This is true often whether a computer makes the judgement or a so-called “expert” does. 
+
+At the end of a talk I gave in a school in south Bristol, one student asked me “what went wrong with the GCSE grades over covid?” I fumbled with my answer, trying to appease and validate a roomful of understandably frustrated bright minds who felt like they had been unfairly penalised by an unseen algorithm in ways they didn’t understand – much in the way teachers in Washington DC  had been penalised by IMPACT. 
+
+On the bus home I thought more about the answer and even made a meme about it. 
+
+There are two answers; one is a nuanced discussion about how predictive models have value but are not the unbiased, omniscient systems they are often lauded and expected to be. Such an answer is explored in books like Weapons of Math Destruction by Cathy O’Neil, Automating Inequality by Virginia Eubanks and Noise by Daniel Kahneman, Oliver Sibony and Cass R. Sunstein, all books which influenced this talk and that I highly recommended to anyone interested in this topic. 
+
+The other answer is a lot more simple: The G in GLMs sometimes stands for “Garbage”. 
```

Comment:
Love it Huw. It reads really nicely and I found it super interesting to read !

Reply:
I take this piece a bit like as an idea for other people wanting to do this kind of thing - in which case it might be nice to have some of your take-aways in terms of advice on running data ethics things in schools. 
Do you think it made students more interested in doing maths? Were there prime ages for this or did it work for everyone?